src/cntk-train/src/main/scala/BrainscriptBuilder.scala

```diff
@@ -74,13 +88,39 @@ class BrainScriptBuilder {
     this
   }
 
+  def textReaderConfig(loc: String, ipstring: String): String = {
+    s"""reader = [ readerType = CNTKTextFormatReader; file = "$loc" ; $ipstring ]"""
+  }
+
+  def parquetReaderConfig(loc: String, ipstring: String): String = {
+    s"""    reader = {
+      |        deserializers = (
+      |                        [
+      |                            type = "DataFrameDeserializer"
+      |                            module = "Cntk.Deserializers.DF"
+      |                            $ipstring
+      |                            hdfs = {
```

Comment:
Nice! is the port number fixed or does it need to be made a parameter.

Reply:
it needs to be fixed according to Tong

Reply:
We can create a constant like CNTKLearner.rpcPortNumber to represent RPC protocol port number and use it to replace the port number in the template.

Reply:
done